client/src/modals/login.jsx

```diff
@@ -1,25 +1,44 @@
 import React, { useState } from 'react';
 import styled from 'styled-components';
 import Modal from 'react-modal';
+import axios from 'axios';
 import PropTypes from 'prop-types';
+import { useHistory } from 'react-router-dom';
 
 import { Button, Tile, Form } from '../styles/globalStyles';
+import cookies from '../cookies';
+import Signup from './signup';
 
 Modal.setAppElement(document.getElementById('app'));
 Modal.defaultStyles.overlay.backgroundColor = 'rgba(0,0,0,0.6)';
 
 export default function Login({ cb = () => {} }) {
   const [showModal, toggleModal] = useState(false);
+  const history = useHistory();
 
   function submitLogin(e) {
     e.preventDefault();
-    const { username, password } = e.target.form;
-    // const data = {
-    //   username: username.value,
-    //   password: password.value,
-    // };
-    username.value = '';
-    password.value = '';
+    const { email, password } = e.target.form;
+    const data = {
+      email: email.value,
+      password: password.value,
+    };
+    axios.post('http://localhost:3010/api/login', data)
+      .then((response) => {
+        console.log(response);
+        const expires = new Date();
+        expires.setMinutes(expires.getMinutes() + 1);
+        cookies.set({
+          userId: response.data.id,
+          expires: expires.toGMTString(),
+        });
+        history.push('/bands');
+      })
+      .catch((error) => {
+        console.log(error);
+      });
+    // email.value = '';
+    // password.value = '';
     cb();
```

Comment:
What does this cb() do?

Reply:
Honestly do not know. I don't remember adding it there, so it's getting snapped out of existence.